packages/playwright-core/src/server/bidi/bidiBrowser.ts

```diff
@@ -126,6 +97,7 @@ export class BidiBrowser extends Browser {
   async doCreateNewContext(options: types.BrowserContextOptions): Promise<BrowserContext> {
     const { userContext } = await this._browserSession.send('browser.createUserContext', {
       acceptInsecureCerts: options.ignoreHTTPSErrors,
+      proxy: getProxyConfiguration(options),
```

Comment:
Does it fix any of the existing tests? If so, can you mention which ones in the PR description?